test/Dialect/VectorOps/ops.mlir

```diff
@@ -33,6 +33,17 @@ func @extractelement(%arg0: vector<4x8x16xf32>) -> (vector<8x16xf32>, vector<16x
   return %1, %2, %3 : vector<8x16xf32>, vector<16xf32>, f32
 }
 
+// CHECK-LABEL: insertelement
+func @insertelement(%a: f32, %b: vector<16xf32>, %c: vector<8x16xf32>, %res: vector<4x8x16xf32>) {
+  //      CHECK: vector.insertelement %{{.*}}, %{{.*}}[3] : vector<8x16xf32> into vector<4x8x16xf32>
```

Comment:
```CHECK: %{{.*}} = ``` at least the first time.

Reply:
I don't think it's relevant here. The LHS of an assignment is printed by the generic printer, which is exercised in core tests, nothing dialect-specific here.

Reply:
It won't print it if the op is a zero result one.

Reply:
It's encoded in tablegen.
We don't have the habit adding extra verification of test for stuff that is covered by tablegen.